src/main/kotlin/org/rust/lang/core/types/RustTraitType.kt

```diff
@@ -9,9 +9,9 @@ import org.rust.lang.core.types.visitors.RustTypeVisitor
  * Though you use the same path to denote both traits and trait objects,
  * only the latter are types.
  */
-class RustTraitObjectType(val trait: RustTraitItemElement) : RustTypeBase() {
+class RustTraitType(val trait: RustTraitItemElement) : RustTypeBase() {
```

Comment:
No, trait object is the correct term here: https://doc.rust-lang.org/reference.html#trait-objects